### PR TITLE
vm-sysroot: add QEMU support (full bootable image)

### PR DESCRIPTION
**Pull request by sleirsgoevy**

### CODE REVIEW

File: vm-sysroot/image/build-full.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+mkdir tmpfs #this is intentional, don't rename
+mount -t tmpfs tmpfs tmpfs
+cp /usr/lib/grub/i386-pc/boot.img tmpfs/
+grub-mkimage -O i386-pc -o tmpfs/core.img -p '(hd0,1)/boot/grub' biosdisk part_msdos ext2 linux normal
+echo '(hd0) ./full.img' > tmpfs/dmap.txt
+grub-bios-setup --device-map tmpfs/dmap.txt -d tmpfs -s full.img
+umount tmpfs
+rmdir tmpfs
+cat hdd.img >> full.img
```
Comment (MikailBag) [inline vm-sysroot/image/build-full.sh]: Is out file location hardcoded?

File: vm-sysroot/image/build-image.sh [hunk 2]
```diff
@@ -6,39 +6,16 @@ if [ "$$" != 1 ] && [ "x$1" != x ]
 then cat >&2 << EOF
 usage: image/build-image.sh
 
-Build the disk image image/hdd.img, using sysroot in ./sysroot. Uses UML to isolate itself.
+Build the disk image image/hdd.img, using sysroot in ./sysroot.
```
Comment (MikailBag) [inline vm-sysroot/image/build-image.sh]: Is this sysroot (and its layout) documented?

File: vm-sysroot/image/build-full.sh (hunk 1 above)
Comment (sleirsgoevy) [reply vm-sysroot/image/build-full.sh]: It is, however it is also hardcoded for the fs-only image

File: vm-sysroot/image/build-image.sh (hunk 2 above)
Comment (sleirsgoevy) [reply vm-sysroot/image/build-image.sh]: This sysroot is just the resulting image's root filesystem. It is build using `./build.sh` or one of its wrappers.

File: vm-sysroot/image/build-full.sh (hunk 1 above)
Comment (sleirsgoevy) [reply vm-sysroot/image/build-full.sh]: EDIT: the sysroot path is not hardcoded, however the image build scripts expect it to be at `./sysroot`

Comment (MikailBag) [reply vm-sysroot/image/build-full.sh]: Then add something like
`--out=image.iso`

**Pull request merged**